src/UI/Factory.php

```diff
@@ -717,4 +717,25 @@ public function legacy($content);
 	 */
 	public function table();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     Alerts inform the user about the state of the system. Such as the successful completion, the need for further
```

Comment:
I don't think its actually the `state of the system` very often, but rather the state of some task or action that the user wanted or will want to perform.

Reply:
I think both is valid and would propose: "Message Boxes inform the user about the state of the system or an ongoing user task."

Reply:
Can you give an instance where the box just informs about the state of the system? Maybe they fit in "Message Boxes inform the user about the state of a task or action the user wanted or will want to perform.". State of the system to me is super broad and I would prefer a more narrow definition (if there is one).

Reply:
E.g. participant data being available in a test. This just informs about the general availability of different interactions. It is not related to a currently performed user interaction (like the other three message types).